dev/migrate.ps1

```diff
@@ -6,12 +6,33 @@
 #  in the future and investigate if we can migrate back.
 # docker-compose --profile mssql exec mssql bash /mnt/helpers/run_migrations.sh @args
 
-docker run `
-  -v "$(pwd)/helpers/mssql:/mnt/helpers" `
-  -v "$(pwd)/../util/Migrator:/mnt/migrator/" `
-  -v "$(pwd)/.data/mssql:/mnt/data" `
-  --env-file .env `
-  --network=bitwardenserver_default `
-  --rm `
-  mcr.microsoft.com/mssql-tools `
-  /mnt/helpers/run_migrations.sh @args
+param([switch]$all = $false, [switch]$postgres = $false, [switch]$mysql = $false)
```

Comment:
doesn't this mean that with no arguments, this script does nothing?

Reply:
With no arguments, migrate.ps1 will operate as it had in the past and just launch the docker run command. 

`if ($all -or (!$postgres -and !$mysql))`

is the same as 

`if ($all -or ($postgres -eq $false -and $mysql -eq $false))`

It would probably be more readable the second way, I'll get it swapped over.

Reply:
I just don't like this being a switch statement. These are all separate toggles, not mutually exclusive ones.